app/assets/stylesheets/common/base/modal.scss

```diff
@@ -704,3 +704,14 @@
     top: 21px;
   }
 }
+
+.json-editor-btn-delete {
+  @extend .btn-danger !optional;
+  @extend .no-text !optional;
+}
+
+.json-editor-btn-collapse {
+  @extend .no-text !optional;
+  @extend .btn-flat !optional;
+  @extend .btn-small !optional;
```

Comment:
Can we double check the syntax here for the flag. I think it should be `!default`?

Reply:
Good catch! 

The `!optional` flag is indeed valid for `@extend`, it lets Sass compilation continue if the extended selector cannot be found. But in this case, those selectors should always be available, so the flag isn't necessary.